src/styles/Blocks/mainBlock.scss

```diff
@@ -0,0 +1,3 @@
+.main {
+  margin: 0 auto;
```

Comment:
let's move this property to .main__cards to center content and pass the tests

Reply:
+ fixed it